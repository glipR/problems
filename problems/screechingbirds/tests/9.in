39 328
51 90 aqvnuiarchbdvw
35 25 whefwpvsx
65 47 wufeswaflnfoik
96 38 ulzoqlq
45 82 rpdlmrtbsyxjkyhktujiumfbv
38 55 lrqlfxaiwqaecamh
26 99 kllvysclxueqnrubyxdrwigzn
87 93 tfpcwytaraxxkodibu
0 46 mfbbdotmyftiuncstbbw
95 20 lmwbdes
77 39 eqrmlqgxlnt
94 84 rfrgadbdjnpuqfauffo
45 52 zrzdhrvaslnbligxvqrxjpoky
70 12 pxdqeqvby
10 3 coujt
84 98 opffsxytxmumnshx
5 43 izjhiwggzmiyfpv
43 15 rdovxdvefuafxlv
7 72 kwxojabdxkkesr
54 47 ggbpfcnusfexlqe
71 64 rqtufbahvibntdhwfumsldsah
35 17 atigmk
99 64 cxnllzrpevhipgma
31 71 jggsydejcyb
65 100 hdslomphxzarmybuvmz
66 2 mmtctxfdmdkitydiyff
55 11 gaieiuomokemhgo
22 55 cjkvnalxqzodfwff
15 47 ldcsyleef
33 89 smigpojrhrfdwj
0 98 pflixqigkgtnhqtjnaanm
92 54 snpwfykpemkln
10 93 cgmkaouqemeaviagetwwl
51 78 cvhraozlydpsjuwsilcx
32 5 askjsvvzv
22 88 jgnkejdyunm
65 67 vgmkpacbyvvukfemucpgsrqg
26 55 xvyrggutzsroqqno
76 2 dkbqbktpkmcavmsldzifsadc