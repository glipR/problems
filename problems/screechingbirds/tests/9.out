mfbbdotmyftiuncstbbw
pflixqigkgtnhqtjnaanm
izjhiwggzmiyfpv
kwxojabdxkkesr
coujt
cgmkaouqemeaviagetwwl
coujt
ldcsyleef
coujt
coujt
coujt
cjkvnalxqzodfwff
jgnkejdyunm
coujt
kllvysclxueqnrubyxdrwigzn
xvyrggutzsroqqno
coujt
coujt
jggsydejcyb
askjsvvzv
smigpojrhrfdwj
coujt
whefwpvsx
atigmk
coujt
askjsvvzv
lrqlfxaiwqaecamh
coujt
askjsvvzv
coujt
rdovxdvefuafxlv
rpdlmrtbsyxjkyhktujiumfbv
zrzdhrvaslnbligxvqrxjpoky
mfbbdotmyftiuncstbbw
coujt
askjsvvzv
izjhiwggzmiyfpv
coujt
aqvnuiarchbdvw
cvhraozlydpsjuwsilcx
coujt
atigmk
askjsvvzv
ggbpfcnusfexlqe
coujt
gaieiuomokemhgo
askjsvvzv
coujt
rdovxdvefuafxlv
whefwpvsx
coujt
ldcsyleef
askjsvvzv
coujt
wufeswaflnfoik
hdslomphxzarmybuvmz
vgmkpacbyvvukfemucpgsrqg
mmtctxfdmdkitydiyff
gaieiuomokemhgo
coujt
askjsvvzv
mmtctxfdmdkitydiyff
atigmk
pxdqeqvby
coujt
mmtctxfdmdkitydiyff
rqtufbahvibntdhwfumsldsah
mmtctxfdmdkitydiyff
askjsvvzv
coujt
rdovxdvefuafxlv
mmtctxfdmdkitydiyff
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
eqrmlqgxlnt
gaieiuomokemhgo
cjkvnalxqzodfwff
askjsvvzv
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
kwxojabdxkkesr
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
xvyrggutzsroqqno
pxdqeqvby
coujt
mmtctxfdmdkitydiyff
askjsvvzv
dkbqbktpkmcavmsldzifsadc
opffsxytxmumnshx
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
whefwpvsx
coujt
atigmk
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
tfpcwytaraxxkodibu
askjsvvzv
coujt
rdovxdvefuafxlv
mmtctxfdmdkitydiyff
gaieiuomokemhgo
dkbqbktpkmcavmsldzifsadc
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
izjhiwggzmiyfpv
mfbbdotmyftiuncstbbw
mmtctxfdmdkitydiyff
snpwfykpemkln
askjsvvzv
dkbqbktpkmcavmsldzifsadc
lrqlfxaiwqaecamh
rfrgadbdjnpuqfauffo
pxdqeqvby
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
lmwbdes
ulzoqlq
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
zrzdhrvaslnbligxvqrxjpoky
coujt
askjsvvzv
mmtctxfdmdkitydiyff
pflixqigkgtnhqtjnaanm
dkbqbktpkmcavmsldzifsadc
cxnllzrpevhipgma
gaieiuomokemhgo
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
ggbpfcnusfexlqe
jggsydejcyb
mmtctxfdmdkitydiyff
askjsvvzv
dkbqbktpkmcavmsldzifsadc
coujt
rdovxdvefuafxlv
atigmk
cgmkaouqemeaviagetwwl
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
pxdqeqvby
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
askjsvvzv
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
ldcsyleef
whefwpvsx
mmtctxfdmdkitydiyff
gaieiuomokemhgo
jgnkejdyunm
dkbqbktpkmcavmsldzifsadc
wufeswaflnfoik
coujt
mmtctxfdmdkitydiyff
askjsvvzv
dkbqbktpkmcavmsldzifsadc
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
lmwbdes
coujt
eqrmlqgxlnt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
askjsvvzv
pxdqeqvby
coujt
rdovxdvefuafxlv
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
atigmk
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
gaieiuomokemhgo
mmtctxfdmdkitydiyff
smigpojrhrfdwj
askjsvvzv
dkbqbktpkmcavmsldzifsadc
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
kllvysclxueqnrubyxdrwigzn
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
rpdlmrtbsyxjkyhktujiumfbv
coujt
askjsvvzv
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
cvhraozlydpsjuwsilcx
pxdqeqvby
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
mmtctxfdmdkitydiyff
gaieiuomokemhgo
cjkvnalxqzodfwff
askjsvvzv
vgmkpacbyvvukfemucpgsrqg
dkbqbktpkmcavmsldzifsadc
coujt
rdovxdvefuafxlv
ulzoqlq
izjhiwggzmiyfpv
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
whefwpvsx
lmwbdes
rqtufbahvibntdhwfumsldsah
coujt
mmtctxfdmdkitydiyff
xvyrggutzsroqqno
dkbqbktpkmcavmsldzifsadc
atigmk
askjsvvzv
mfbbdotmyftiuncstbbw
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
aqvnuiarchbdvw
pxdqeqvby
coujt
mmtctxfdmdkitydiyff
askjsvvzv
dkbqbktpkmcavmsldzifsadc
gaieiuomokemhgo
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
mmtctxfdmdkitydiyff
snpwfykpemkln
dkbqbktpkmcavmsldzifsadc
askjsvvzv
lrqlfxaiwqaecamh
coujt
rdovxdvefuafxlv
ggbpfcnusfexlqe
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
zrzdhrvaslnbligxvqrxjpoky
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
kwxojabdxkkesr
mmtctxfdmdkitydiyff
askjsvvzv
dkbqbktpkmcavmsldzifsadc
pxdqeqvby
coujt
atigmk
mmtctxfdmdkitydiyff
gaieiuomokemhgo
dkbqbktpkmcavmsldzifsadc
lmwbdes
eqrmlqgxlnt
mmtctxfdmdkitydiyff
ldcsyleef
dkbqbktpkmcavmsldzifsadc
coujt
askjsvvzv
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
wufeswaflnfoik
whefwpvsx
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
mmtctxfdmdkitydiyff
askjsvvzv
dkbqbktpkmcavmsldzifsadc
coujt
rdovxdvefuafxlv
cxnllzrpevhipgma
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
hdslomphxzarmybuvmz
gaieiuomokemhgo
pxdqeqvby
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
askjsvvzv
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
atigmk
ulzoqlq
coujt
mmtctxfdmdkitydiyff
askjsvvzv
dkbqbktpkmcavmsldzifsadc
jggsydejcyb
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
lmwbdes
coujt
mmtctxfdmdkitydiyff
gaieiuomokemhgo
dkbqbktpkmcavmsldzifsadc
izjhiwggzmiyfpv
askjsvvzv
rfrgadbdjnpuqfauffo
pxdqeqvby
coujt
rdovxdvefuafxlv
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
tfpcwytaraxxkodibu
mmtctxfdmdkitydiyff
dkbqbktpkmcavmsldzifsadc
coujt
opffsxytxmumnshx
mmtctxfdmdkitydiyff
askjsvvzv
